### PR TITLE
Add ATO token metadata

### PR DESCRIPTION
**Pull request by linhai0212**
# Add ATO token metadata

NEP-141:  https://explorer.near.org/accounts/atocha-token.near
Aurora ERC-20 (BlockScout): https://explorer.mainnet.aurora.dev/address/0x2ab98d9eA81aF20037AF1a4f43CC3e6977545840/
Aurora ERC-20 (Aurorascan): https://aurorascan.dev/address/0x2ab98d9eA81aF20037AF1a4f43CC3e6977545840

### CODE REVIEW

File: tokens/ato.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "aurora_address": "0x2ab98d9ea81af20037af1a4f43cc3e6977545840",
```
Comment (sept-en) [inline tokens/ato.json:2]:
```suggestion
    "ethereum_address": "",
    "aurora_address": "0x2ab98d9ea81af20037af1a4f43cc3e6977545840",
```

File: tokens/ato.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "aurora_address": "0x2ab98d9ea81af20037af1a4f43cc3e6977545840",
+    "near_address": "atocha-token.near",
+    "name": "Atocha Coin",
+    "symbol": "ATO",
+    "decimals": 18
```
Comment (sept-en) [inline tokens/ato.json:6]:
```suggestion
    "decimals": 18,
    "icon": "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 136.88 136.88'%3E%3Cdefs%3E%3Cstyle%3E.cls-1%7Bfill:gold;%7D%3C/style%3E%3C/defs%3E%3Ctitle%3Eatocha logo%3C/title%3E%3Cg id='Layer_2' data-name='Layer 2'%3E%3Ccircle class='cls-1' cx='68.44' cy='68.44' r='68.44'/%3E%3C/g%3E%3Cg id='Layer_1' data-name='Layer 1'%3E%3Cpath d='M73.86,82q0,12.2,0,24.41c0,1.27.24,1.49,1.47,1.17,1.08-.29,2.13-.68,3.18-1a11.58,11.58,0,0,0,1.5-.59c1.41-.72,2.84-1.4,4.18-2.22a25.59,25.59,0,0,0,5.59-4.64c1.36-1.52,2.54-3.2,3.78-4.82.33-.43.19-.72-.35-.71-.86,0-1.72.13-2.58.22-.49,0-1,.13-1.47.19a.39.39,0,0,1-.48-.37l-.33-2a1.55,1.55,0,0,0-.07-.42,5.78,5.78,0,0,1-.33-2.13c-.09-1-.25-1.94-.34-2.92-.07-.78.05-.84.83-.94,1.59-.19,3.18-.44,4.77-.65L96,84.21l1.14-.15,4.87-.74c.27,0,.54-.05.8-.09l2.18-.35a2.75,2.75,0,0,1,.55-.08.47.47,0,0,1,.35.17,9.85,9.85,0,0,1,.44,1.1c.24.69.46,1.39.69,2.08a2.69,2.69,0,0,1,.09.28c.17.83.33,1.66.51,2.49.26,1.14.54,2.27.81,3.4.31,1.31.6,2.63.93,3.93.19.75.45,1.49.65,2.24s.45,1.81.66,2.71a.4.4,0,0,1-.14.34,3.39,3.39,0,0,1-.81.35c-1.36.36-2.72.69-4.09,1l-2.51.65c-.63.17-.77.08-.92-.54q-.39-1.59-.82-3.17c0-.14-.21-.34-.32-.34a.91.91,0,0,0-.59.24c-.29.33-.48.74-.75,1.08-.93,1.18-1.9,2.33-2.83,3.51a30,30,0,0,1-4.6,4.3,39.4,39.4,0,0,1-3.74,2.58,26.59,26.59,0,0,1-2.71,1.59c-1.08.53-2.2,1-3.32,1.42s-2,.78-3.07,1.11c-.71.23-1.45.37-2.17.55-.2.05-.4.14-.6.21a4.51,4.51,0,0,1-.44.16c-.61.15-1.23.3-1.85.43a.62.62,0,0,0-.53.7c0,.78,0,1.57-.06,2.34A5.28,5.28,0,0,1,73,122.2a3.88,3.88,0,0,1-1.52,1.32,12.57,12.57,0,0,1-1.54.71,1.24,1.24,0,0,1-.78.07,5.86,5.86,0,0,1-3.73-2.92,8,8,0,0,1-.4-2.62,10,10,0,0,1,0-1.28.67.67,0,0,0-.61-.74c-.75-.14-1.5-.26-2.24-.44s-1.57-.44-2.35-.68c-.5-.15-1-.28-1.5-.47-1.63-.61-3.28-1.19-4.89-1.86A15.12,15.12,0,0,1,51.17,112c-1.38-.89-2.78-1.77-4.1-2.76-1-.76-1.92-1.68-2.88-2.52a36.33,36.33,0,0,1-4.81-5.68c-.58-.77-1.1-1.58-1.68-2.35a.81.81,0,0,0-.6-.29c-.16,0-.37.25-.43.43-.31,1-.57,2.07-.86,3.11q-.19.7-.4,1.39c-.11.35-.39.34-.66.28-.58-.13-1.16-.28-1.73-.43-1.62-.42-3.24-.84-4.86-1.28a6.55,6.55,0,0,1-1-.4.34.34,0,0,1-.12-.3c.21-1,.41-1.93.66-2.89.33-1.24.72-2.47,1-3.72.38-1.48.72-3,1.08-4.47.22-.89.46-1.78.68-2.68.18-.72.34-1.45.52-2.17s.32-1.23.52-1.83.38-.55.88-.42a6.74,6.74,0,0,0,.92.15c1.2.15,2.39.28,3.59.44l5.6.75,4.61.57c.85.11,1.69.25,2.53.36.3,0,.47.19.44.51,0,.14,0,.29,0,.43-.13,1.24-.22,2.49-.4,3.72s-.45,2.38-.66,3.58c-.06.34-.23.47-.55.42l-2.14-.35a2.2,2.2,0,0,0-.47,0c-.4,0-.57.22-.42.59a7.92,7.92,0,0,0,.71,1.34,24.94,24.94,0,0,0,3.52,4.21c.91.88,1.82,1.76,2.78,2.57A19,19,0,0,0,56.73,105c1.11.51,2.22,1,3.37,1.47s2.38.8,3.58,1.18a2,2,0,0,0,.51.05.47.47,0,0,0,.52-.4,2.91,2.91,0,0,0,.07-.55c.07-3.6.18-7.19.2-10.79.05-8.32.06-16.64.08-25,0-4.54,0-9.07,0-13.61,0-.9-.2-1.09-1.1-1.09-5.35,0-10.7,0-16,0a13.51,13.51,0,0,1-2.78-.39,1.31,1.31,0,0,1-.73-.7,3.45,3.45,0,0,1-.33-1.35c0-1.43,0-2.87,0-4.31a3.45,3.45,0,0,1,.31-1.1,1,1,0,0,1,.95-.63c1.3-.11,2.6-.24,3.91-.26,3.63,0,7.25,0,10.88-.05H64c.66,0,1-.27,1-.93,0-.21,0-.43,0-.64q0-5.42,0-10.84a1.43,1.43,0,0,0-.75-1.34,10.49,10.49,0,0,1-3.55-3.56,7.57,7.57,0,0,1-1.29-3.7c-.06-1.19,0-2.39,0-3.58a8.35,8.35,0,0,1,1.21-3.38,9.5,9.5,0,0,1,4-3.76,6.53,6.53,0,0,1,2.68-1c.73-.08,1.44-.3,2.17-.34a6.16,6.16,0,0,1,1.42.22c.13,0,.25.08.37.09a10.35,10.35,0,0,1,5,2.35,7.62,7.62,0,0,1,2.18,2.8c.46.93.8,1.93,1.17,2.9a2.12,2.12,0,0,1,.1.67c0,.48,0,1,0,1.45A8.12,8.12,0,0,1,79,28.43a12.31,12.31,0,0,1-2.17,3.63,5.8,5.8,0,0,1-1.93,1.57,1.68,1.68,0,0,0-1.06,1.74c0,3.58,0,7.17,0,10.75,0,.17,0,.34,0,.51,0,.62.29.86.92.88.26,0,.51,0,.77,0,4.12,0,8.25,0,12.37,0,1.76,0,3.52.14,5.28.16a1.82,1.82,0,0,1,1.68,1.85c0,1.54,0,3.07,0,4.61a2.33,2.33,0,0,1-.57,1.44,1.35,1.35,0,0,1-.7.41,20.07,20.07,0,0,1-5.18.34c-4.51,0-9,0-13.53,0-.91,0-1.07.16-1.07,1.06Q73.86,69.72,73.86,82ZM69.37,30.45a5.74,5.74,0,0,0,5.71-5.64A5.82,5.82,0,0,0,69,19.18c-2.52.05-5.11,2.83-5.07,5.71A5.6,5.6,0,0,0,69.37,30.45Z' transform='translate(-2.21 -2.5)'/%3E%3Cpath d='M83.15,80a20,20,0,0,1-2.69-.37,7.33,7.33,0,0,1-4.34-3,7.7,7.7,0,0,1-1-6.24,7.78,7.78,0,0,1,3.76-5,10.25,10.25,0,0,1,6.38-1.51,8.1,8.1,0,0,1,6.17,3.4,7.65,7.65,0,0,1,1.35,6.22,6.46,6.46,0,0,1-2.14,3.46,12.27,12.27,0,0,1-3.46,2.21A10.64,10.64,0,0,1,83.15,80Zm3-5.66a6.69,6.69,0,0,0,0-.85,22.5,22.5,0,0,0-1.17-4.77,1.83,1.83,0,0,0-1.48-1.31c-1.26-.13-1.92,1-1.8,2.19A23.72,23.72,0,0,0,82.34,73a13.07,13.07,0,0,0,.86,2.39,1.48,1.48,0,0,0,2.33.56A1.81,1.81,0,0,0,86.14,74.32Z' transform='translate(-2.21 -2.5)'/%3E%3Cpath d='M46.15,75.81a3.87,3.87,0,0,1,2-3.6,8.91,8.91,0,0,1,5.06-1.32,5.59,5.59,0,0,0,.6,0,1,1,0,0,0,1-1,10,10,0,0,0-.27-2.56,1.33,1.33,0,0,0-2.34-.54c-.57.55-1,1.21-1.57,1.79a2.56,2.56,0,0,1-2.22.94,1.47,1.47,0,0,1-1.21-2,3.93,3.93,0,0,1,2-2.09,13.26,13.26,0,0,1,2.57-1.08,10.31,10.31,0,0,1,5.13-.27,7.35,7.35,0,0,1,2.93,1.36,3.38,3.38,0,0,1,1.38,2.45c.07,1.27.08,2.54.15,3.81.06,1,.16,2,.26,3a1.23,1.23,0,0,0,.88,1.05c.67.27.74.46.57,1.14a3.8,3.8,0,0,1-2,2.35,4.38,4.38,0,0,1-3.68.45A3.3,3.3,0,0,1,56.24,79c-.74-.68-.82-.75-1.62-.19a6.21,6.21,0,0,1-4.69,1A4.21,4.21,0,0,1,46.83,78,3.1,3.1,0,0,1,46.15,75.81Zm7.7-2.91a2,2,0,0,0-1.17,3.43,1.22,1.22,0,0,0,1.74-.24,2.9,2.9,0,0,0,.26-2.62A.76.76,0,0,0,53.86,72.9Z' transform='translate(-2.21 -2.5)'/%3E%3C/g%3E%3C/svg%3E",
    "reference": "",
    "reference_hash": ""

```

**Review by sept-en — CHANGES_REQUESTED** [review]

**Comment by linhai0212** [comment]
Merged thanks

**Review by sept-en — APPROVED** [review]
Thanks for your PR! The metadata on Aurora will be updated at the beginning of next week.

**Pull request merged**